package.json

```diff
@@ -22,7 +22,8 @@
         "build:ts:watch": "wireit",
         "build:types": "wireit",
         "build:watch": "wireit",
-        "custom-element-json": "lerna exec --ignore \"{@spectrum-web-components/{base,bundle,clear-button,close-button,modal,iconset,shared,opacity-checkerboard,styles,custom-vars-viewer,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- cem analyze --config ../../custom-elements-manifest.config.js --packagejson",
+        "changeset-publish": "changeset publish --message \"chore: release new versions #publish\" --force-publish",
+        "custom-element-json": "changeset exec --ignore \"{@spectrum-web-components/{base,bundle,clear-button,close-button,modal,iconset,shared,opacity-checkerboard,styles,custom-vars-viewer,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc}\" -- cem analyze --config ../../custom-elements-manifest.config.js --packagejson",
```

Comment:
Related to my previous comment about the valid command line options, I think this script will need to be refactored as it seems to be using some of Lerna's functionality, which will no longer work for Changesets.

Maybe the things here will need to be invoked via Yarn?

Reply:
Yes the above can be updated to use yarn like
```
"custom-element-json": "yarn workspaces foreach --exclude \"@spectrum-web-components/{base,bundle,clear-button,close-button,modal,iconset,shared,opacity-checkerboard,styles,custom-vars-viewer,eslint-plugin},stylelint-header,@swc-react/*,documentation,example-project-rollup,example-project-webpack,swc-templates,@types/swc\" run cem:analyze --config ../../custom-elements-manifest.config.js --packagejson",
```